src/main/java/io/fabric8/maven/docker/access/NetworkCreateConfig.java

```diff
@@ -0,0 +1,26 @@
+package io.fabric8.maven.docker.access;
+
+import org.json.JSONObject;
+
+public class NetworkCreateConfig {
+    final JSONObject createConfig = new JSONObject();
+    final String name;
```

Comment:
wonder whether it would make sense to support more of the options as described in https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/#create-a-network ?